packages/fluentui/react-northstar/src/themes/teams-dark/components/Menu/menuVariables.ts

```diff
@@ -33,7 +33,6 @@ export const menuVariables = (siteVars: any): Partial<MenuVariables> => ({
   activeUnderlinedBorderBottomColor: siteVars.colors.white,
   activeUnderlinedPrimaryColor: siteVars.colors.brand[400],
 
-  verticalBackgroundColorFocus: siteVars.colors.grey[550],
```

Comment:
This was the only place where the variable was set. Does it make sense to keep the variable or should we remove it completely?

Reply:
good point, I think it should be removed